cluster-bootstrap/src/main/resources/reference.conf

```diff
@@ -95,6 +95,13 @@ akka.management {
 
       # Timeout for getting a reply from the service-discovery subsystem
       resolve-timeout = 3 seconds
+
+      # Does a succcessful response have to be received by all contact points.
+      # Used by the LowestAddressJoinDecider
+      # Can be set to false in environments where old contact points may still be in service discovery
+      # or when using local discovery and cluster formation is desired without starting all the nodes
+      # Required-contact-point-nr still needs to be met
```

Comment:
So, with this changed to `false`, say it would discover 10 nodes, but not get response from 5, then as long as `required-contact-points = 2` is fulfilled it will go ahead and form a cluster?

What is a scenario where "old contact points may still be in service discovery"?

Reply:
That's right.

>What is a scenario where "old contact points may still be in service discovery"?

For instance when they are in a DNS cache